public/locales/ru-ru.json

```diff
@@ -878,201 +626,181 @@
     "Recent": "Последние",
     "Most chats": "Больше всего чатов",
     "Least chats": "Меньше всего чатов",
-    "Most tokens": "Больше всего токенов",
-    "Least tokens": "Меньше всего токенов",
-    "Random": "Случайно",
-    "Toggle character grid view": "Изменить вид грида персонажей",
-    "Bulk_edit_characters": "Массовое редактирование персонажей",
-    "Bulk select all characters": "Массовый выбор всех персонажей",
-    "Bulk delete characters": "Массовое удаление персонажей",
-    "popup_text_save": "Сохранить",
-    "popup_text_yes": "Да",
-    "popup_text_no": "Нет",
-    "popup_text_cancel": "Отмена",
-    "popup_text_import": "Импортировать",
-    "Advanced Defininitions": "Расширенное описание",
+    "Back": "Назад",
     "Prompt Overrides": "Индивидуальные промпты",
-    "(For Chat Completion and Instruct Mode)": "(для Chat Completion и режима Instruct)",
+    "(For OpenAI/Claude/Scale APIs, Window/OpenRouter, and Instruct Mode)": "(для API OpenAI/Claude/Scale, Window/OpenRouter, а также режима Instruct)",
```

Comment:
This text was pruned from the JSON file because it no longer exists in the app. So I guess you have to run the tool again to identify and remove all of the ghost i18n strings.